lib/line_drive.ex

```diff
@@ -34,6 +34,25 @@ defmodule LineDrive do
     Tesla.client(middleware)
   end
 
+  def build_client(refresh_token, client_id, client_secret, base_url) do
```

Comment:
Should the other function that uses an api key be updated to the same name or stay separate?

Reply:
I'm open to other ways of doing it, but it feels like a sufficiently different thing. `client` only ever returns a client struct, where `build_client` is doing more stuff that can fail, and returns a tuple.